README.md better explanation of Parsing Zone Files

taich001 opened the pull request:
Add imports
change print() cause of python3
```
>>> import blockstack_zones
>>> import json
>>> zone_file_object = parse_zone_file(zone_file)
>>> print(json.dumps(zone_file_object, indent=4, sort_keys=True))
```

jaywink committed:
Run stage1 of futurize

jaywink committed:
Run stage2 of futurize

jaywink committed:
Make tests pass

jaywink committed:
Some minimal cleaning, pep8fying and removing unused imports

jaywink committed:
Rename to blockstack-zones-py3

Remove python2 compatibility

jaywink committed:
Add .idea to .gitignore

jaywink committed:
Fix pypi badge on the readme, clarify this is a fork

claassistantio commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/blockstack/zone-file-py?pullRequest=25) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/blockstack/zone-file-py?pullRequest=25) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/blockstack/zone-file-py?pullRequest=25) it.</sub>

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/blockstack/zone-file-py?pullRequest=25) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/blockstack/zone-file-py?pullRequest=25) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/blockstack/zone-file-py?pullRequest=25) it.</sub>

philipcristiano commented:
This would be awesome to get merged, I ran into the import issue on py3!

@jcnelson mind taking a look?

jcnelson commented:
Hey @philipcristiano, @taich001,

Thank you for sending this PR!  Unfortunately, we have no plans to move to Python 3, and are in the process of removing all of the Python modules in this organization (we're moving to Typescript and Rust).  However, please feel free to maintain a Python 3 fork if you find it useful :+1:

aredwood committed:
Semicolon Support

Resolved an issue where txt records would terminate at the first semicolon.

This prevents DMARC records, which have semicolons from properly parsing.

- Tests have also been added to reflect this change.

jaywink committed:
Merge pull request #1 from aredwood/bugfix/semicolon-txt-records

Semicolon Support

jaywink committed:
Bump version to 0.19.1
